service-api/src/main/java/greencity/dto/CreateAddressRequestDto.java

```diff
@@ -71,4 +75,34 @@ public class CreateAddressRequestDto {
     @Pattern(regexp = STREET_REGEXP, message = STREET_VALIDATION_MESSAGE)
     @NotEmpty(message = notEmptyValidationMessage)
     private String streetEn;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
+        CreateAddressRequestDto that = (CreateAddressRequestDto) o;
+        return Objects.equals(this.districtEn.trim().toLowerCase(), that.districtEn.trim().toLowerCase()) &&
+            Objects.equals(this.district.trim().toLowerCase(), that.district.trim().toLowerCase()) &&
+            Objects.equals(this.regionEn.trim().toLowerCase(), that.regionEn.trim().toLowerCase()) &&
+            Objects.equals(this.region.trim().toLowerCase(), that.region.trim().toLowerCase()) &&
+            Objects.equals(this.houseNumber, that.houseNumber) &&
+            Objects.equals(this.entranceNumber, that.entranceNumber) &&
+            Objects.equals(this.houseCorpus, that.houseCorpus) &&
+            Objects.equals(this.coordinates, that.coordinates) &&
+            Objects.equals(this.city.trim().toLowerCase(), that.city.trim().toLowerCase()) &&
+            Objects.equals(this.cityEn.trim().toLowerCase(), that.cityEn.trim().toLowerCase()) &&
+            Objects.equals(this.street.trim().toLowerCase(), that.street.trim().toLowerCase()) &&
+            Objects.equals(this.streetEn.trim().toLowerCase(), that.streetEn.trim().toLowerCase());
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(this.districtEn.trim().toLowerCase(), this.district.trim().toLowerCase(),
+            this.regionEn.trim().toLowerCase(), this.region.trim().toLowerCase(), this.houseNumber.trim().toLowerCase(),
```

Comment:
You use this.houseNumber for `equals` and ` this.houseNumber.trim().toLowerCase()` for `hashCode`
Please use the same values for `equals` and `hashCode`
Please check all `equals` and `hashCode` methods